styles/styles.css

```diff
@@ -0,0 +1,1029 @@
+    height: 1px;
+    margin-bottom: 10px;
+    margin-left: 20px;
+    width: 95.7vw;
+}
+
+/* FOOTER */
+
+.top-footer {
+    display: flex;
+    justify-content: space-between;
+    margin-left: 40px;
+}
+
+.top-footer-left {
+    align-items: baseline;
+    display: flex;
+    flex-direction: row;
+}
+
+.top-footer-left img {
+    color: red;
+    height: 36px;
+    margin-bottom: 15px;
+    margin-right: 15px;
+    width: 236px;
+}
+
+.top-footer-left p {
+    color: #666666;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 12px;
+    letter-spacing: normal;
+    line-height: 12px;
+}
+
+.footer-icons {
+    display: flex;
+    flex-direction: row;
+    margin-right: 30px;
+}
+
+.footer-icons img {
+    height: 25px;
+    margin: 7px;
+    width: 25px;
+}
+
+.line-9 {
+    background-color: lightgray;
+    height: 1px;
+    margin-bottom: 10px;
+    margin-left: 35px;
+    width: 94vw;
+}
+
+.bottom-footer {
+    display: flex;
+    justify-content: space-between;
+}
+
+.list-links {
+    margin-left: 35px;
+    width: 666px;
+}
+
+.list-links li, .list-links a, .country-links li, .country-links a {
+    color: #666666;
+    display: inline;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 14px;
+    line-height: 15px;
+    margin: 0 3px;
+}
+
+.location-links {
+    display: flex;
+    flex-direction: row;
+    margin-right: 30px;
+}
+
+.location-links p {
+    color: #666666;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 14px;
+    font-weight: bold;
+    line-height: 15px;
+    margin-right: 5px;
+    margin-top: 5px;
+}
+
+.country-links {
+    height: 54px;
+    margin-right: 5px;
+    width: 253px;
+}
+
+
+/* Media queries */
+
+/* Large devices (desktops, 992px and up) */
+
+@media (max-width: 1200px) and (min-width: 976px) {
+    .newsletter-info p {
+        font-size: 10px;
+        margin-bottom: 5px;
+    }
+
+    .newsletter-info a {
+        font-size: 13px;
+    }
+}
+
+@media (max-width: 992px) {
+     
+    .editor-section {
+        grid-template-columns: 1fr 1fr;
+    }
+
+    .ed-1, .ed-3 {
+        grid-column: 1 / span 1;
+    }
+
+    .ed-2, .ed-4 {
+        grid-column: 2 / span 1;
+    }
+
+    .location-links {
+        display: none;
+    }
+
+}
+
+/* Medium devices (tablets, 768px and up) */
+@media (max-width: 768px) { 
+    .in-the-magazine, .editor-section {
+        display: grid;
+        grid-template-columns: 1fr;
+    }
+    
+    .article-1, .article-2, .article-3, .article-4 {
+        grid-column: 1 / span 1;
+    }
+
+    .ed-1, .ed-2, .ed-3, .ed-4 {
+        grid-column: 1 / span 1;
+    }
+
+ }
+
+/* Small devices (landscape phones, 576px and up) */
+@media (max-width: 576px) { 
+    .sponsor {
+        text-align: center;
+    }
+
+    .sponsored-insight {
+        align-content: center;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+    }
+
+    .category-link-2 {
+        display: none;
```

Comment:
`SUGGESTION`

Consider using a Bootstrap class name for this selector element and in all other occurrences.